pkgs/development/python-modules/pytest-workflow/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  jsonschema,
+  pytest,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-workflow";
+  version = "2.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "LUMC";
+    repo = "pytest-workflow";
+    rev = "v${version}";
+    hash = "sha256-ztR4TW41qVAnCLXPUsWaiMuD1ESL0Kvd5KAsOf8tcE0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    jsonschema
+    pytest
```

Comment:
`pytest` should go to `buildInputs` to make overriding easy.
See other pytest plugins.

Reply:
> `pytest` should go to `buildInputs` to make overriding easy. See other pytest plugins.

This is not resolved yet.